AISKU/src/AISku.ts

```diff
@@ -259,8 +261,40 @@ export class AppInsightsSku implements IApplicationInsights {
                     _self.emptyQueue();
                     _self.pollInternalLogs();
                     _self.addHousekeepingBeforeUnload(_self);
+
+                    _addUnloadHook(onConfigChange(cfgHandler, () => {
+                        if (!_throttleMgr){
+                            _throttleMgr = new ThrottleMgr(_core);
+                        }
+                        //  && !_config.throttleMgrCfg.disabled
+                        if (_throttleMgr && !_throttleMgr.isReady() && _config.extensionConfig && _config.extensionConfig[_cfgSyncPlugin.identifier]) {
+                            // set ready state to true will automaatically trigger flush()
+                            _throttleMgr.onReadyState(true);
+                        }
+
+                        if (!_iKeySentMessage){ //should not send again
+                            if (!_config.connectionString && !isFeatureEnabled("disableIkeyDeprecationMessage", _config)) {
+                                _throttleMgr.sendMessage( _eInternalMessageId.InstrumentationKeyDeprecation, "Instrumentation key support will end soon, see aka.ms/IkeyMigrate");
+                                _iKeySentMessage = true;
+                            }
+                        }
+                       
+                        if (!_cdnSentMessage){
+                            if (sdkSrc && sdkSrc.indexOf("az416426") != -1 && !isFeatureEnabled("disableCdnDeprecationMessage", _config)) {
+                                _throttleMgr.sendMessage( _eInternalMessageId.CdnDeprecation, "Support for this domain will end soon, https://aka.ms/JsActiveCdn");
+                                _cdnSentMessage = true;
+                            }
+                        }
+                       
+                        if (!_sdkVerSentMessage){
+                            if (parseInt(_snippetVersion) < 6 && !isFeatureEnabled("disableSnippetVersionUpdateMessage", _config)) {
+                                _throttleMgr.sendMessage( _eInternalMessageId.SnippetUpdate, "Snippet ver is updated, see https://aka.ms/SnippetVer");
```

Comment:
"A newer Sdk Loader is available, see https://aka.ms/xxxxx" or
"An updated Sdk Loader is available, see ...."